### PR TITLE
Input handle improvements

### LINKED ISSUE #64: String split error.
On PRIVMSG behavior:
The user here is called `tester` and will have his name truncated if the column is right after it.
```
PRIVMSG tester: Hey
401 me :teste :No such nick/channel
PRIVMSG testerr: Hey
//Success
```

### LINKED ISSUE #65: Whitespaces handling
What should happen with this kind of input:

`KICK    #lol   me  :   nooooo`

Is this acceptable? The current output is:

`403 tester  :No such channel`

### PR DESCRIPTION
**Pull request by NotCampeur**
Closes #64
Closes #65 
Basic check and tweak of every command input parsing.
Few fixes has been implemented here as well.

### CODE REVIEW

File: srcs/commands/parsing.cpp
Comment (jfremond) [inline srcs/commands/parsing.cpp:1]: The msg.text variable is not empty, it actually contains two characters

**Review by Bee-Have — APPROVED** [review]
All is good, a few errors have been corrected with @NotCampeur.

File: srcs/commands/parsing.cpp
Comment (Bee-Have) [reply srcs/commands/parsing.cpp:1]: msg.text is a variable used in all the commands, I am unsure what you are referring to. Could you elaborate ?

Comment (jfremond) [reply srcs/commands/parsing.cpp:1]: When testing, I found that the first condition to exit the `parsing_cmds` function was not true `if (msg.text.empty() == true) return ;`
I then printed the length of the variable to see why I could send empty commands and I found that the length of msg.text was 2 (I assume those the characters are "\r\n"). (I also assume that msg.text contains the full line sent before any parsing (PASS test, NICK nickname, and so on...))

Comment (NotCampeur) [reply srcs/commands/parsing.cpp:1]: Indeed, the condition has been improved to:
`if (msg.text.empty() == true || msg.text.find_first_not_of(" \n\r\t") == std::string::npos)`
Thanks

**Pull request merged**